src/librustc_back/target/apple_ios_base.rs

```diff
@@ -79,7 +79,8 @@ fn target_cpu(arch: Arch) -> String {
         Arm64 => "cyclone",
         I386 => "yonah",
         X86_64 => "core2",
-    }.to_string()
+    }
+    .to_string()
```

Comment:
This is one of the more common complaints at the moment. It's not ideal, but it's not likely to get fixed in the very short term (it's sub-optimal, but I don't think it is wrong or super-ugly and isn't a bug). I'd suck this up for now, on the understanding that we'll get to it at some point in the near-ish future and it will get prettier.

(btw, the reason it's not trivial to fix is that if there is more than one method call or the call is more complicated it can be desirable to do it this way, then consistency comes into play).